react/week4/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, createContext } from "react";
+import "./App.css";
+import Header from "./components/Header";
+import InputForm from "./components/InputForm";
+import GetUsersData from "./components/GetUsersData";
+import MyRepos from "./components/MyRepos";
+import "./App.css";
+
+export const userListContext = createContext();
+
+function App() {
+  const [input, setInput] = useState("");
+  const [user, setUser] = useState([]);

```

Comment:
We should always be as consistent and accurate with our variables naming. If the variable holds an array, it means it's a list of elements (users in this case), meaning we use plural
```suggestion
  const [users, setUsers] = useState([]);

```